docs/crashlytics/android-setup.md

```diff
@@ -55,7 +55,18 @@ apply plugin: 'com.google.firebase.crashlytics'
 // ..
 ```
 
-## 4. (Optional) Enable Crashlytics NDK reporting
+## 4. Add line for implementation
+
+Inside the `android/build.gradle` file, in the dependencies add the following implementations:
+
+```
+implementation 'com.google.firebase:firebase-crashlytics'
+implementation 'com.google.firebase:firebase-analytics'
+implementation 'com.google.firebase:firebase-iid:21.1.0'
```

Comment:
firebase iid is no longer a valid dependency to use externally, it was deprecated with v21 and it is discontinued upstream  https://firebase.google.com/support/release-notes/android#iid_v21-0-0

You should not use it